opentreemap/treemap/css/sass/partials/pages/_admin.scss

```diff
@@ -256,7 +257,8 @@
             border-bottom: 1px solid #EEE;
 
             &.active {
-                background-color: #F7F3DF;
+                background-color: $warning-color;
+                background-color: rgba($warning-color, .75);
```

Comment:
Yeah I did. line 260 is a fallback for ie8 which I wasn't sure if we supported which I just asked Deb. She said not to worry about it but it also can't hurt to have it in place.